app/views/anthologies/_searched_documents.erb

```diff
@@ -23,13 +23,11 @@
                 <span class='badge'><%= all_documents_count %></span> All
               <% end %>
             </li>
-            <% if @anthology.owned_tags.count > 0 || current_user.admin? %>
```

Comment:
Check my logic @sarastanton. This should be: 

`<% if @anthology.user == current_user || current_user.admin? %>`

Otherwise, the tab never shows for non-admins, even if they own the anthology, unless there's already a folder created, which there won't ever be if you can't see the tab.

Actually, this one line might be the whole PR. 🤔

Reply:
@jamiefolsom Agreed! I tested this out locally and it seems to do what we're wanting it to do (i.e. non-admin users can manage folders for _only_ their own anthologies, and admin users can manage folders for all anthologies they have access to). I'll update the PR 👍

Reply:
Actually, do we need that `owned_tags.count > 0` too, so that non-owner, non-admins don't see the tab? What do you think?

Reply:
@sarastanton let's do try that additional condition. So the statement would be:

`<% if @anthology.user == current_user || owned_tags.count > 0 || current_user.admin? %>`

Reply:
Correction! 

`<% if @anthology.user == current_user || @anthology.owned_tags.count > 0  || current_user.admin? %>`

Reply:
> Actually, do we need that `owned_tags.count > 0` too, so that non-owner, non-admins don't see the tab? What do you think?

@jamiefolsom  Wouldn't this make it so that non-owner, non-admin users _do_ see the tab, if there is at least 1 existing folder?

Reply:
@sarastanton hm, yes, I think so, but how else would a non-admin, non-owner see existing folders, if that tab is not visible to them? Or would they just be shown as tags in that folders column? Maybe that's good enough, if so -- would you let me know if that's a coherent UX from your pt of view, and I'll ask if that works?